core/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1882,7 +1882,7 @@ public static DateTimeZone randomDateTimeZone() {
     /**
      * Returns path to a random directory that can be used to create a temporary file system repo
      */
-    public Path randomRepoPath() {
+    public static Path randomRepoPath() {
```

Comment:
Why did you choose to make `#randomRepoPath()` static? As you pass in the test case instance to `SnapshotSharedTest`anyway? I assume you decided that because the implementation only relies on class state, not instance state.